params/config.go

```diff
@@ -1043,6 +1050,13 @@ func (c *ChainConfig) checkCompatible(newcfg *ChainConfig, headNumber *big.Int,
 	if isForkBlockIncompatible(c.MergeNetsplitBlock, newcfg.MergeNetsplitBlock, headNumber) {
 		return newBlockCompatError("Merge netsplit fork block", c.MergeNetsplitBlock, newcfg.MergeNetsplitBlock)
 	}
+	if c.Optimism != nil && newcfg.Optimism != nil {
+		oldBlock := uint64ptr2Big(c.Optimism.SoulGasTokenBlock)
```

Comment:
I’m curious - why do we use a fork block number for SGT, but a fork timestamp for L2 BLOBs? Is there a specific reason for choosing different fork triggers in these cases?

Reply:
Good question, I think it's something we can unify after the merge.

Reply:
Let us create a ticket to track this.

Reply:
Tracking in https://github.com/QuarkChain/op-geth/issues/16.